packages/camera/camera/lib/camera.dart

```diff
@@ -170,7 +57,7 @@ class CameraPreview extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return controller.value.isInitialized
-        ? Texture(textureId: controller._textureId)
+        ? Texture(textureId: controller._cameraId)
```

Comment:
The whole `CameraPreview` widget has become obsolete and should be removed. The widget is replace by the `buildView` method.

B.T.W. the `buildView` method should be renamed to `buildCameraPreview`